test/index.ts

```diff
@@ -318,6 +318,24 @@ client.on('messageCreate', async (msg: Message) => {
     } else {
       msg.channel.send(msg.author.avatarURL())
     }
+  } else if (msg.content.startsWith('!roleIcon') === true) {
+    const size = msg.mentions.roles.size
+    const role = msg.mentions.roles.first()
+    const icon = role?.roleIcon()
+    if (size === 0) {
+      msg.channel.send('no role mentioned')
+    } else {
+      if (icon === undefined) {
+        msg.channel.send('no icon')
+      } else {
+        msg.channel.send(icon)
+      }
+    }
+  } else if (msg.content === '!roleSmile') {
+    const role = await msg.guild?.roles.get('834440844270501888')
+    if (role !== undefined) {
+      role.edit({ unicode_emoji: '😀' })
```

Comment:
Option names shouldn't be snake case

Reply:
Do these get automatically transformed into snake case when sending the request or do I need to change something for that manually?